client/android/app/src/main/java/com/snapscreen/MainApplication.java

```diff
@@ -24,6 +25,7 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
               new MainReactPackage(),
+            new ReactNativeConfigPackage(),
```

Comment:
Maybe we should use an autoformatter for the Java code as well to avoid indentation problems like this?